.github/workflows/tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  tests:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", pypy-3.7]
+        # It is difficult to install libxml2 and libxslt development packages on Windows.
+        # https://www.lfd.uci.edu/~gohlke/pythonlibs/ distributes a wheel, but the URL changes.
+        exclude:
+          - os: windows-latest
+            python-version: pypy-3.7
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install system libraries
+      if: contains(matrix.os, 'ubuntu') && contains(matrix.python-version, 'pypy')
+      run: |
+        sudo apt-get update
+        sudo apt-get install libxml2-dev libxslt-dev
```

Comment:
Which packages require these?

Reply:
Scrapy. I followed your example here: https://github.com/scrapy/scrapyd-client/commit/279330eae3e05646f7905aeb2edab4777c136b46 but actually let's try without this and see what's up.

Reply:
looks like it works fine without this step, so I removed it https://github.com/scrapy/scrapyd/pull/412/commits/61bbff591d746679f817e463eaebdce92a27888d